ckanext/who_afro/schemas/indicator.yaml

```diff
@@ -0,0 +1,109 @@
+scheming_version: 2
+dataset_type: indicator
+name: Indicator
+about: A customized dataset schema for WHO AFRO CKAN deployment
```

Comment:
I think this could be more detailed

Reply:
I think this PR is a step towards a data catalogue implementation, but only a small one - the focus is on getting programme, type and country working so that we can implement the home page design from lewis. All other catalgoue changes I am leaving for a further PR e.g. we have talked about implementing DCAT.